backend/src/main/scala/bloop/Compiler.scala

```diff
@@ -938,49 +971,52 @@ object Compiler {
           backgroundTasksWhenNewSuccessfulAnalysis
             .map(f => f(clientClassesDir, clientReporter, clientTracer))
         val allClientSyncTasks = Task.gatherUnordered(successBackgroundTasks.toList).flatMap { _ =>
-          // Only start these tasks after the previous IO tasks in the external dir are done
-          val firstTask = updateExternalClassesDirWithReadOnly(
```

Comment:
Should this be removed? Will be sync the needed files to the client directory?

Reply:
Yes, but in that case do not want to use the readonlyDir (previous compilation), and instead use just newDir (current compilation)